### DISCUSSION TITLE
Modifying Sonic’s Sprites

### DISCUSSION THREAD

**Discussion by WimpyKidFan2007** [Techokami/SonicWorldsNext#139] [Q&A]
How do I modify Sonic's sprites in Sonic Worlds Next?

**Comment by DimensionWarped** [comment]
An open ended question. The easiest way is to just replace frames on the Sonic png file directly by hand... that said, if the sprite you are trying to replace it with is differently scaled or has different frame counts for its animations then you'll need to learn how to use the sprite editing systems in Godot.

That said, this is all general Godot stuff that you should be learning before trying to use a complex template like Worlds to make a game (not saying it's anything other than a great way to learn Godot, but maybe learn to walk before you start trying to replace sprites with dozens of animations in them).

**Reply by WimpyKidFan2007** [reply]
Can't I just modify the sprites in Aseprite?

**Reply by DimensionWarped** [reply]
Sure, if you want to just draw over the existing frames that's fine. Just be aware that there are invisible borders to the sprite frames that you might want to use helpers or something to show if your own sprites are going to be larger than the existing ones in one dimension or another.
